env/development

```diff
@@ -1,10 +1,10 @@
 PTERO_PETRI_AMQP_HOST=localhost
-PTERO_PETRI_AMQP_PORT=5672
 PTERO_PETRI_AMQP_VHOST=/
 PTERO_PETRI_AMQP_RETRY_DELAY=30
 PTERO_PETRI_AMQP_CONNECTION_ATTEMPTS=40
 PTERO_PETRI_AMQP_PREFETCH_COUNT=10
 PTERO_PETRI_AMQP_HEARTBEAT_INTERVAL=600
```

Comment:
Are all these other AMQP env variables used? perhaps we should make all of them RABBITMQ_* or whatever.  Doesn't have to be this PR though.